services/meta/association.md

```diff
@@ -81,10 +111,65 @@ _IMPORTANT_
   "type": "evt.association.report",
   "val_t": "object",
   "val": {
-    "group": "group_1",
+    "group": 3,
     "member": ["2_1", "3_1"],
   },
   "props": null,
   "tags": null
 }
+
+{
+  "serv": "association",
+  "type": "evt.groups_info.report",
+  "val_t": "object",
+  "val": [
+    {
+      "commands": [
+        {
+          "class": 128,
+          "command": 3
```

Comment:
The only thing that I'm not sure of here are those two properties... I'm pretty sure, translation - so 128 - COMMAND_CLASS_X_Y would be more readable, at least for z-wave technicals.

For example, for myself to understand those I also have to open up the specification to see what kind of CC is that, but if it was named it would be easier.

Reply:
@BudzikM How does it fit ZIgbee associations?